### DISCUSSION TITLE
Ahoi-DTU keine Verbindung zu HM-350

### DISCUSSION THREAD

**Discussion by NewBee51** [tbnobody/OpenDTU#1393] [Q&A]
Hallo Zusammen,

ich habe mir eine Ahoi - DTU mit dem Wemos D1 ESP32 gebaut. Das Aufspielen der Software hat gut funktioniert, die DTU ist auch im WLAN erreichbar und kann konfiguriert werden. 

Leider lässt sich mein WR HM-350 nicht verbinden, die Seriennummer habe ich korrekt eingegeben. 

Was kann ich sonst noch prüfen, testen ?

Viele Grüße

NewBee51

**Comment by NewBee51** [comment]
![image](https://github.com/tbnobody/OpenDTU/assets/146536215/21b2dc17-785b-45d7-8e3d-808cf55c26bb)

**Comment by NewBee51** [comment]
Das WLAN-Modul sollte das richtige sein (mit dem +) und im Status steht "is connected"

**Comment by tbnobody** [comment]
Es macht ggf. mehr Sinn diese Frage im Repository von AhoyDTU zu stellen und nicht hier bei OpenDTU. --> Unterschiedliche Software

**Reply by NewBee51** [reply]
oh, sorry - da bin ich aus versehen im falschen Repository gelandet. Danke für den Hinweis.
